docs/reference/run.md

```diff
@@ -772,45 +772,161 @@ In addition to `--privileged`, the operator can have fine grain control over the
 capabilities using `--cap-add` and `--cap-drop`. By default, Docker has a default
 list of capabilities that are kept. The following table lists the Linux capability options which can be added or dropped.
 
-| Capability Key | Capability Description |
-| :----------------- | :---------------| :-------------------- |
```

Comment:
@henrysher You don't have to use ``<table>`` to fix this, it's *just* about the aligment of these two lines (up). If you make it so it looks like the following, it should work :

```Markdown
| Capability Key | Capability Description |
|----------------|------------------------|
```